### PR TITLE
Added catching OperationalError exception for django dbs check

### PR DESCRIPTION
**Pull request by kosmos342**
Tested locally with postgresql and mysql, when db was stopped, `/status/` endpoint returned 500 error with `OperationalError`

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # [Changelog](https://github.com/yola/healthcheck)
 
+## 0.1.2
+* Added cathing OperationalError exception for django dbs check.
```
Comment (ekoval) [inline CHANGELOG.md]: catChing

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: healthcheck/checks.py [hunk 2]
```diff
@@ -101,8 +101,12 @@ def items(self):
         return connections.all()
 
     def check_item(self, connection):
-        connection.ensure_connection()
-        db_ok = connection.is_usable()
+        from django.db.utils import OperationalError
+        try:
+            connection.ensure_connection()
+            db_ok = connection.is_usable()
```
Comment (serhii-karelov) [inline healthcheck/checks.py]: I would move it out of `try:` block. Looks like  `connection.is_usable()` will never raise such kind of error.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: healthcheck/checks.py (hunk 2 above)
Comment (kosmos342) [reply healthcheck/checks.py]: moved

File: tests/test_checks.py [hunk 3]
```diff
@@ -183,6 +189,43 @@ def test_error_when_checking_if_files_exist(self, stat_mock):
...
+        connections_mock.all.return_value = [connection_mock]
+        check = DjangoDBsHealthCheck()
+        check.run()
+        self.assertTrue(check.is_ok)
+        self.assertEqual(check.details, {'db_name': 'ok'})
+        connection_mock.ensure_connection.assert_called_once_with()
+        connection_mock.is_usable.assert_called_once_with()
+
+
+    @patch('django.db.connections')
```
Comment (snitch) [inline tests/test_checks.py]: E303 too many blank lines (2)

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**